scenarios/kubernetes_kops_aws.py

```diff
@@ -168,6 +187,17 @@ def main(args):
         '--kops-ssh-key=%s' % aws_ssh,
     ]
 
+    if args.build is not None:
```

Comment:
I don't understand how I can get both args.build = None and args.build = '' given that this flag is required. How will this happen?